vendor/aquascope/wwd.yaml

```diff
@@ -1,4 +1,4 @@
-name: Flood Sensor
+name: FLOLWE01 - Flood Sensor
```

Comment:
Why?

Reply:
Cause theres multiple devices named flood sensor, and ben is getting a duplicate error: https://github.com/TheThingsNetwork/lorawan-devices/pull/749#issuecomment-1966305882

Reply:
The uniqueness should be on vendor name + device name, not just device name

Reply:
Ok i understand, but then we have a problem. If you look at that comment from Ben you can see that airtable can not link the name and vendor together, so it will count devices with the same name as a duplicate. I dont know if you can fix this in airtable somehow @benolayinka but we need to figure out how we're going to do that.

Reply:
How about adding simply another column with the full name, just for enforcing uniqueness?

Reply:
Made a column with device name + vendor, that way we can be fully sure its unique.